plugins/pencilblue/templates/angular/admin/content/articles/article_form.html

```diff
@@ -35,6 +36,21 @@
 			});
 		};
 
+		$scope.generateUrl = function() {
+			$scope.article.url = $scope.article.headline.replace(/\s+/g, '-').toLowerCase();
+			if($scope.article.url){
+				$scope.urlGenerated = true;
+			} else {
```

Comment:
Will do the url check but what do you mean by "We'll never have this hit, if the headline is valid."?